drivers/include/periph/rtc.h

```diff
@@ -147,6 +148,27 @@ void rtc_tm_normalize(struct tm *time);
  */
 int rtc_tm_compare(const struct tm *a, const struct tm *b);
 
+/**
+ * @brief Convert time struct into timestamp.
+ *
+ * @pre   The time structs @p a and @p b are assumed to be normalized.
+ *        Use @ref rtc_tm_normalize to normalize a struct tm that has been
+ *        manually edited.
+ *
+ * @param[in] t       The time struct to convert
+ *
+ * @return            elapsed seconds since `RIOT_EPOCH`
+ */
+uint32_t rtc_mktime(struct tm *t);
+
+/**
+ * @brief Converts an RTC timestamp into a  time struct.
+ *
+ * @param[in] time    elapsed seconds since `RIOT_EPOCH`
```

Comment:
I know, it's a bit nitpicky but it looks better in source code when it is aligned with the parameter in next line
```suggestion
 * @param[in]  time   elapsed seconds since `RIOT_EPOCH`
```